andela/ah-magnificent6
# #159965290 Upgrade Author's Haven app from Django 1.10 to Django2.1

#### What does this PR do?
- Upgrade Author's Haven app to Django 2.1 from Django 1.10

#### Description of Task to be completed?
- Upgrade Django from version 1.10 to version 2.1
- Upgrade Django Rest Framework to a version compatible with Django 2.1 - Upgraded to version 3.8.2
- Upgrade Django Extensions to a version compatible with Django 2.1 - Upgraded to version 2.1.2

#### How should this be manually tested?
- Clone the repo and cd into the project folder
- Checkout into the `ch-upgrade-django1.10-to-django2.1-159965290` branch
- Create a virtual environment by running `python3 -m venv venv`
- Activate the virtual enviroment by running `source venv\bin\activate`
- Run `pip install -r requirements.txt`
- Run `django-admin version` It should show `2.1`

#### What are the relevant pivotal tracker stories?
[Upgrade Authors Haven Django app from Django 1.10 to Django 2.1](https://www.pivotaltracker.com/n/projects/2192364/stories/159965290)
